lms/djangoapps/grades/context.py

```diff
@@ -5,18 +5,19 @@
 
 from openedx.core.djangoapps.content.block_structure.api import get_course_in_cache
 
+from .course_grade import CourseGrade
 from .scores import possibly_scored
 
 
-def grading_context_for_course(course_key):
+def grading_context_for_course(course):
     """
     Same as grading_context, but takes in a course key.
```

Comment:
Docstring should be updated to reflect change in parameter.

Reply:
done. will merge in a separate PR.